cmd/platform/message_export.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+package main
+
+import (
+	l4g "github.com/alecthomas/log4go"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
+var exportCmd = &cobra.Command{
+	Use:   "export",
+	Short: "Export data from Mattermost",
+}
+
+var actianceExportCmd = &cobra.Command{
+	Use:     "actiance",
+	Short:   "Actiance XML",
+	Long:    "Export data in the Actiance XML format.",
+	Example: "  export actiance",
+	RunE:    actianceExportCmdF,
+}
+
+func init() {
+	exportCmd.AddCommand(
+		actianceExportCmd,
+	)
+}
+
+func actianceExportCmdF(cmd *cobra.Command, args []string) error {
```

Comment:
This CLI command starts a new instance of the export job and blocks until it completes. Right now, the job doesn't actually complete, likely because I haven't set the config values for it to reasonable defaults.